vnext/Shared/OInstance.cpp

```diff
@@ -385,6 +389,24 @@ InstanceImpl::InstanceImpl(
           m_devSettings->jsiRuntimeHolder =
               std::make_shared<Microsoft::JSI::ChakraRuntimeHolder>(m_devSettings, m_jsThread, nullptr, nullptr);
           break;
+        case JSIEngineOverride::V8NodeAapi: {
+#if defined(USE_V8)
+          std::unique_ptr<facebook::jsi::PreparedScriptStore> preparedScriptStore = nullptr;
+
+          char tempPath[MAX_PATH];
+          if (GetTempPathA(MAX_PATH, tempPath)) {
```

Comment:
>GetTempPathA [](http://example.com/codeflow?start=14&length=12)

Could someone go through all *A() apis, and replace them with proper Unicode *W() Apis in this file?

Reply:
This would be an issue for the V8 cases.
The underlying API expects 8-bit strings. `GetTempPathW` would send 16-bit strings.
I have opened an issue for follow up.

Reply:
Yup, unfortunately Windows will have to pay some utf8 to utf16 converson penalty because it doesn't support utf8 natively.